### PR TITLE
Allow unverified connection

### PR DESCRIPTION
**Pull request by elferherrera**
Allow unverified connection

### CODE REVIEW

**Comment by elferherrera** [comment]
@regadas do you think we can merge this one as well?

File: src/ls/driver.ts [hunk 1]
```diff
@@ -27,12 +27,18 @@ export default class TrinoDriver
       return this.connection;
     }
 
+    var server = this.credentials.server
+    if (this.credentials.port) {
+      server = server + ":" this.credentials.port
+    }
```
Comment (regadas) [inline src/ls/driver.ts:33]: Hi @elferherrera here, I think I would like to keep server unchanged. You can write it as `location:port` ... similar to what you do through the trino cli.

**Review by regadas — COMMENTED** [review]
closed in favor of #382

**Comment by regadas** [comment]
Thanks for this @elferherrera ❤️

**Pull request closed without merging**